### PR TITLE
Display the screenshots checkbox on all platforms.

### PR DESCRIPTION
**Pull request by mstange**
r? @brennie 

Not sure why it decided to reformat unrelated code...

### CODE REVIEW

**Comment by julienw** [comment]
> Not sure why it decided to reformat unrelated code...

My guess is that `yarn.lock` (that I generated once) defines a newer version of prettier than the one that we used at first. Because this isn't checked by CI or by automatic scripts at push time we never reformatted our files with this version.

File: popup.js [hunk 1]
```diff
@@ -245,12 +247,7 @@ window.onload = async () => {
  */
 async function setupFeatureCheckbox(name) {
   const platform = await browser.runtime.getPlatformInfo();
-  if (name == 'screenshots') {
-    if (platform.os !== 'mac' && platform.os !== 'linux') {
-      document.querySelector('#screenshots').style.display = 'none';
```
Comment (julienw) [inline popup.js:250]: I think you can also remove the id in https://github.com/firefox-devtools/Gecko-Profiler-Addon/blob/master/popup.html#L136

**Review by julienw — APPROVED** [review]
lgtm, thanks!

File: popup.js (hunk 1 above)
Comment (mstange) [reply popup.js:250]: Good catch, done.

**Comment by julienw** [comment]
@mstange Ah I realize that in `background.js` we define the default value for this checkbox, which is `true` on mac but `false` elsewhere. What's the status in the various platforms, should we enable it by default everywhere?

**Pull request merged**